packages/bierzo-wallet/src/routes/addresses/index.e2e.spec.ts

```diff
@@ -85,7 +84,7 @@ withChainsDescribe("E2E > Receive Payment route", () => {
     }, 35000);
   });
 
-  describe("Starnames tab", () => {
+  describe.skip("Starnames tab", () => {
```

Comment:
Is this ok, or you forgot to remove?

Reply:
Since they need some rewrite I prefer to do it as part of #1039.